molo/profiles/migrations/0019_userprofile_uuid.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-02-02 11:04
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import uuid
+
+
+def create_uuid(apps, schema_editor):
+    UserProfile = apps.get_model('profiles', 'UserProfile')
+    for user in UserProfile.objects.all():
+        user.uuid = uuid.uuid4()
+        user.save()
```

Comment:
Please use `update_fields` as shown in the [docs](https://docs.djangoproject.com/en/1.11/howto/writing-migrations/#migrations-that-add-unique-fields)